crypto/evp/encode.c

```diff
@@ -115,6 +146,11 @@ int EVP_ENCODE_CTX_num(EVP_ENCODE_CTX *ctx)
     return ctx->num;
 }
 
+void evp_encode_ctx_set_flags(EVP_ENCODE_CTX *ctx, uint64_t flags)
+{
+    ctx->flags = flags;
+}
```

Comment:
It's not impossible to imagine that this might go public at some point, and in such case I'd argue in favour of using types that don't require additional headers such as unsigned int. In other words I'd like to suggest to consider declaring flags as unsigned int, here and in structure...

Reply:
Changed to unsigned int.